spec/services/migrations/map_copyright_migrator_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Migrations::MapCopyrightMigrator do
+  describe ".call" do
+    context "when a map is restricted" do
+      it "updates the rights statement to be \"in copyright\"" do
+        stub_ezid(shoulder: "99999/fk4", blade: "123456")
+        scanned_map = FactoryBot.create_for_repository(:complete_campus_only_scanned_map)
+        raster_map = FactoryBot.create_for_repository(:complete_campus_only_raster_resource)
+        vector_map = FactoryBot.create_for_repository(:complete_campus_only_vector_resource)
```

Comment:
I think it would be good to add one geo resource that's open and then an expectation that its rights statement has not changed.

Reply:
Good point, @hackartisan . I just pushed a commit that adds this expectation.